### PR TITLE
[new release] moonpool (0.1)

### PR DESCRIPTION
**Pull request by c-cube**
Pools of threads supported by a pool of domains

- Project page: <a href="https://github.com/c-cube/moonpool">https://github.com/c-cube/moonpool</a>

##### CHANGES:

initial release

### CODE REVIEW

**Comment by c-cube** [comment]
I wonder if some of the failures are a weird race condition between the main library's installation, and mdx?

**Comment by haochenx** [comment]
>I wonder if some of the failures are a weird race condition between the main library's installation, and mdx?

Probably adding `(deps %{lib:moonpool}` to the [`(mdx)` stanza](https://github.com/c-cube/moonpool/blob/bc949727d6c72ffc1891a2a66b02f6dcb4127be3/dune#L5) may solve this problem.

ref: https://dune.readthedocs.io/en/stable/dune-files.html#:~:text=(deps%20%3Cdeps%2Dconf%20list%3E)%20to%20specify%20the%20dependencies%20of%20your%20documentation%20code%20blocks.

**Comment by c-cube** [comment]
Actually seems like `(mdx)` can take a `(libraries …)` field now, which should be even better. Thanks for the pointer @haochenx :)

File: packages/moonpool/moonpool.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+maintainer: ["Simon Cruanes"]
+authors: ["Simon Cruanes"]
+license: "MIT"
+tags: ["thread" "pool" "domain"]
+homepage: "https://github.com/c-cube/moonpool"
+bug-reports: "https://github.com/c-cube/moonpool/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.0"}
+  "either"
```
Comment (haochenx) [inline packages/moonpool/moonpool.0.1/opam:12]:
```suggestion
  "either" {>= "1.0.0"}
```

please consider add a lower bound

Comment (c-cube) [reply packages/moonpool/moonpool.0.1/opam]: why? it's the lowest release on opam?

**Review by haochenx — APPROVED** [review]
Thanks! All look to me.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**